sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubWebJobsBuilderExtensions.cs

```diff
@@ -103,7 +103,7 @@ internal static void ConfigureInitialOffsetOptions(EventHubOptions options)
                         catch (FormatException fe)
                         {
                             string message = $"{nameof(EventHubOptions)}:{nameof(InitialOffsetOptions)}:{nameof(InitialOffsetOptions.EnqueuedTimeUtc)} is configured with an invalid format. " +
```

Comment:
I don't think the code in try would ever throw a formatException. We don't parse anything there since the change.

Reply:
Hah you're right. I guess we will have to rely on the default exception that comes from the configuration parsing.

Reply:
actually this can still throw an InvalidOperationException if no date was specified.

Reply:
we shouldn't try to grab it in that case

Reply:
Yeah, I am adding separate handling.